met/src/basic/vx_util/command_line.cc

```diff
@@ -750,8 +750,6 @@ for (j=0; j<N; ++j)  {
 
       }
 
-      return ( j );
```

Comment:
I don't know the logic here, but did you intend to remove this return?

Reply:
The line 753 can not be executed because of the if statement above: continue or exit( 1 )
```
      if ( AllowUnrecognizedSwitches )  continue;
      else {
         mlog << Error << "\nCommandLine::next_option() -> "
              << "unrecognized command-line switch \"" << args[j] << "\"\n\n";
         exit ( 1 );
      }
      return ( j );  <-- not reachable
```